tensorflow/python/data/experimental/kernel_tests/counter_test.py

```diff
@@ -17,17 +17,19 @@
 from __future__ import division
 from __future__ import print_function
 
+from absl.testing import parameterized
+
 from tensorflow.python.data.experimental.ops import counter
 from tensorflow.python.data.kernel_tests import test_base
 from tensorflow.python.data.ops import dataset_ops
+from tensorflow.python.framework import combinations
 from tensorflow.python.framework import dtypes
-from tensorflow.python.framework import test_util
 from tensorflow.python.platform import test
 
 
-@test_util.run_all_in_graph_and_eager_modes
-class CounterTest(test_base.DatasetTestBase):
+class CounterTest(test_base.DatasetTestBase, parameterized.TestCase):
 
+  @combinations.generate(test_base.default_test_combinations())
```

Comment:
optional, parametrize the test with start, step, and expected values: `(3, 4, [3, 7, 11]), (0, -1, [0, -1, -2])`

Reply:
Good suggestion. Changed it.